src/cmd-koji-upload

```diff
@@ -192,6 +192,13 @@ class Build(_Build):
 
         return fname, True
 
+    def get_rpms_list(self):
+        """
+        Translate commitmeta.json rpms into a json list
+        Returns the json rpms list
+        """
+        return json.dumps(self.commit["rpmostree.rpmdb.pkglist"])
```

Comment:
I think we'll need to generate a more structured list according to the example `components` key here - https://docs.pagure.org/koji/content_generator_metadata/#example-metadata-json

The scratch code I worked up:

```python
with open("builds/latest/x86_64/commitmeta.json") as f:
    cm = json.load(f)

components = []

for rpm in cm['rpmostree.rpmdb.pkglist']:
    entry = { "type": "rpm",
        "name": rpm[0],
        "version": rpm[2],
        "release": rpm[3],
        "epoch": rpm[1],
        "arch": rpm[4],
        "sigmd5": None,
        "signature": None }
    components.append(entry)
```